config.go

```diff
@@ -33,11 +33,11 @@ var (
 	}
 	kafkaCompression       = "none"
 	kafkaBatchNumMessages  = "10000"
-	kafkaSslClientCertFile = ""
-	kafkaSslClientKeyFile  = ""
-	kafkaSslClientKeyPass  = ""
-	kafkaSslCACertFile     = ""
-	kafkaSslValidation     = true
+        kafkaSslClientCertFile = ""
+        kafkaSslClientKeyFile  = ""
+        kafkaSslClientKeyPass  = ""
+        kafkaSslCACertFile     = ""
+        kafkaSslValidation     = true
```

Comment:
Fix indentation

```suggestion
      kafkaSslClientCertFile = ""
      kafkaSslClientKeyFile  = ""
      kafkaSslClientKeyPass  = ""
      kafkaSslCACertFile     = ""
      kafkaSslValidation     = true
```

Reply:
I think that setting `kafkaSslValidation = false` by default would be more consistent (ssl is disabled by default). This implies moving https://github.com/Telefonica/prometheus-kafka-adapter/pull/38/files#diff-7ddfb3e035b42cd70649cc33393fe32cR45-R47 to the "`if`" block before https://github.com/Telefonica/prometheus-kafka-adapter/pull/38/files#diff-7ddfb3e035b42cd70649cc33393fe32cR39

Reply:
done